makesig.py

```diff
@@ -136,7 +135,7 @@ def process(start_at = MAKE_SIG_AT['fn']):
 	fm = currentProgram.getFunctionManager()
 	fn = fm.getFunctionContaining(currentAddress)
 	if not fn:
-		raise Exception("Not in a function")
+		printerr("Not in a function")
```

Comment:
Raising an exception here (and subsequently breaking out of the script) makes sense, since this script doesn't generate signatures for arbitrary bytes.

Reply:
Fixed in https://github.com/nosoop/ghidra_scripts/pull/5/commits/b3f0a71cbebf5a0140f61b36ae79f40dba971936 The logic wasn't corrected here from another version. Much of this PR may come down to personal preferrence on what we want to see in these scenarios. An exception being thrown generates a lot of other information which didn't seem as helpful as the error message in correcting user behavior.